shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -966,6 +973,9 @@ - (void)dispatchTouches:(NSSet*)touches
     }
   }
 
+  // Activate or pause touch rate correction according to the touches when user is interacting.
```

Comment:
"touch rate" is a little ambiguous. maybe "frame rate during touch events"?

Reply:
I have changed it to `delivery frame rate of touch`, which seems to be more accurate.😄